source/dubregistry/mirror.d

```diff
@@ -53,21 +53,31 @@ nothrow {
 	try {
 		DbPackage[] packs;
 		URL url;
-		auto path = NativePath(fileOrUrl);
+
+		const path = NativePath(fileOrUrl);
 
 		string source_text;
 
-		if (path.existsFile) source_text = path.readFileUTF8;
+		if (path.existsFile) {
+			logInfo("Using local file at %s", path);
+			source_text = path.readFileUTF8;
+		}
 		else {
 			url = URL(fileOrUrl);
+
+			const uri = url ~ InetPath("api/packages/dump");
+
+			logInfo("Downloading packages from %s", uri);
+
 			source_text = requestHTTP(url ~ InetPath("api/packages/dump"))
 				.bodyReader
 				.readAllUTF8;
 		}
 
+		logInfo("Download complete. Begin deserializing");
```

Comment:
Isn't that a bit too verbose ? How long does this operation take ?

Reply:
Oops, might have left some info there for when I was debugging. I had some timeouts when fetching the registry. I had to download the dump separately (not zipped) and use that for some reason.

I can remove it now however

Reply:
Removed